### PR TITLE
Add quiet mode

### PR DESCRIPTION
**Pull request by arai-a**
Just added quiet mode, as a preparation for https://github.com/binast/binjs-ref/issues/166

### CODE REVIEW

File: src/bin/decode.rs [hunk 1]
```diff
@@ -41,18 +46,24 @@ fn main() {
     // Setup.
     let printer = Shift::new();
 
-    println!("Reading.");
+    if !quiet {
```
Comment (Yoric) [inline src/bin/decode.rs]: Maybe make this a macro?

**Comment by Yoric** [comment]
I don't know the best practices here, is it to remove all output or to redirect it to stderr?

**Comment by arai-a** [comment]
given progress is not error, stdout should be fine on non-quiet mode.
on quiet mode, no progress should be printed.

actually, I think even error should not be printed on quiet mode (tell the result only with exit status), but I'm not sure how to handle that in rust's `expect()` etc

**Comment by Yoric** [comment]
Ok, then land as you see fit :)

**Pull request merged**